pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -83,6 +83,18 @@ var typeBuiltinsHaveUnderscore = map[oid.Oid]struct{}{
 	types.AnyTuple.Oid():    {},
 }
 
+// UpdateCommands Enumeration to match update operations in postgres
+const (
+	UpdateCommand = 2 + iota
+	InsertCommand
+	DeleteCommand
+)
+
+var (
+	zero = tree.NewDInt(0)
```

Comment:
nit: zero is very vague in this case. i would do something like

```
type UpdatableCommand tree.DInt

const (
   nonUpdatableCommand UpdatableCommand = 0
   updateCommand UpdatableCommand = 2
// = 3, = 4 ...., all lowercase

   allUpdatableEvents = ( 1 << ... )
)
```

Reply:
Done.

I kept nonUpdatableEvents and allUpdatableEvents as "variables" to have them as pointers of tree.DInt so I can return them as Datums.